Automattic/wp-calypso
# Pattern Assembler - Blank canvas CTA replace image

#### Proposed Changes

* Replaced the Blank canvas CTA image
* Remove the old image from `static/images/`
* Move the new image within the `design-picker` component folder `assets/images/`
* Add SVG type declaration

#### Testing Instructions

- Click on the Calypso Live (direct link) in the comment below.
- Type the URL `/setup/designSetup?siteSlug={Site slug}&flags=signup/design-picker-pattern-assembler` in the Calypso Live domain
- When the category `Show all` is selected
- Check that that Blank canvas CTA has the [new image](https://user-images.githubusercontent.com/1881481/190088953-f9b828ce-1776-4b59-a478-de3a47adab47.png)

<img width="1501" alt="Screen Shot 2565-09-14 at 15 26 02" src="https://user-images.githubusercontent.com/1881481/190102062-6798e7d9-8e6d-4d0d-b5c7-eec869380ffc.png">

#### Pre-merge Checklist






Related to #67786
